ci-operator/step-registry/upi/gcp/post/upi-gcp-post-commands.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export HOME=/tmp
+export GOOGLE_CLOUD_KEYFILE_JSON="${CLUSTER_PROFILE_DIR}/gce.json"
+export CLOUDSDK_ACTIVE_CONFIG_NAME=${GOOGLE_CLOUD_KEYFILE_JSON}
+
+if [ -f ${SHARED_DIR}/metadata.json ]; then
+  INFRA_ID=$(jq -r .infraID ${SHARED_DIR}/metadata.json)
+  set +e
+  gcloud deployment-manager deployments delete -q ${INFRA_ID}-{worker,control-plane,bootstrap,security,infra,vpc}
```

Comment:
Not wrapping this in some higher-level binary means that we can't add it to periodic deprovisioning -- is that reasonable? why?

Reply:
Can you elaborate on `wrapping this in some higher-level binary`?